docs-src/security-model.md

```diff
@@ -68,9 +68,10 @@ readable by most roles, so instead we'll focus on write access for this model.
 | Application Admins | No | In Specified Namespaces | In Specified Namespaces |
 | Application Developers | No | No | In Specified Namespaces |
 
-## Namespace Restrictions
-The extra configuration options are not possible to control with RBAC. Instead,
-they will be controlled with configuration fields on GatewayClasses:
+## GatewayClass Namespace Limitations 
+Kubernetes RBAC only allows us to grant access to create Gateways as a whole,
+not limit creation to specific GatewayClasses. A field on GatewayClass allows
```

Comment:
@robscott you're referencing the `allowedGatewayNamespaceSelector` field, correct?

Reply:
Thanks for catching this! This whole section was out of date - I've updated it here but will also need to reconcile that whenever https://github.com/kubernetes-sigs/service-apis/pull/263 merges.